### ISSUE TITLE
[DocBug] Users buying Multiple Plans of same SaaS app through in-App.

### ISSUE THREAD

**Issue by Konapalli1508** [MicrosoftDocs/msteams-docs#8216]
[Enter feedback here]
The current in-app purchase system only allows ISVs to purchase a single plan, and if they want to buy a different plan or additional plans, they are not able to do so through in-app purchases. Instead, they are required to delete or change their existing plan before they can purchase a new one.
However, this limitation does not apply to the App Source, where ISVs can purchase multiple plans for the same user within their tenant. 

---
#### Document Details

⚠ *Do not edit this section. It is required for learn.microsoft.com ➟ GitHub issue linking.*

* ID: 5a3c7a65-c0e1-ac5b-1027-d11edf172092
* Version Independent ID: 5a3c7a65-c0e1-ac5b-1027-d11edf172092
* Content: [Test preview for monetized apps - Teams](https://learn.microsoft.com/en-us/microsoftteams/platform/concepts/deploy-and-publish/appsource/prepare/test-preview-for-monetized-apps)
* Content Source: [msteams-platform/concepts/deploy-and-publish/appsource/prepare/Test-preview-for-monetized-apps.md](https://github.com/MicrosoftDocs/msteams-docs/blob/main/msteams-platform/concepts/deploy-and-publish/appsource/prepare/Test-preview-for-monetized-apps.md)
* Service: **msteams**
* GitHub Login: @v-ypalikila
* Microsoft Alias: **v-ypalikila**

**Comment by Nivedipa-MSFT** [comment]
@Konapalli1508 - Thanks for reporting your issue. We will check and update you soon.

**Comment by Konapalli1508** [comment]
Yes, I confirm @Nivedipa-MSFT . You can add the mentioned information in the document.

**Comment by Nivedipa-MSFT** [comment]
@Konapalli1508 - We are checking this with engineering team to update the content in document. We will let you know the updates.
